trecs/models/recommender.py

```diff
@@ -234,7 +251,7 @@ def __init__(  # pylint: disable=R0913,R0912,R0915
         assert self.users and isinstance(self.users, Users)
         self.num_users = num_users
         self.num_items = num_items
-        self.num_items_per_iter = num_items_per_iter
+        self.set_num_items_per_iter(num_items_per_iter)
```

Comment:
Perhaps this could be `set_num_recs_per_iter` since `set_num_items_per_iter` could be confused with the total number of items in the system? What do y'all think?

Reply:
I don't have a particular preference here, but two thoughts:
- `num_items_per_iter` does refer to the total number of items in the system when it is equal to `all` afaik
- `num_recs_per_iter` might sound like the number of recommended items, when this could actually include both recommendations and random items.
I can't think of a solution that takes into account these two points, though. Thoughts?

Reply:
Hmm. Those are good points, and this seems to be tricky. This isn't critical, so I'm fine with leaving it if that seems to be the best option.

Reply:
Yeah, I agree with both of you. I have no strong feelings so I'm fine leaving as is for now!